stubtest.allowlist.to_review

```diff
@@ -0,0 +1,25 @@
+PyQt5.QtBluetooth.QBluetoothDeviceInfo.Field.__new__
+PyQt5.QtCore.QAbstractItemModel.CheckIndexOption.__new__
+PyQt5.QtCore.QByteArray.Base64DecodingStatus.__new__
+PyQt5.QtCore.QCalendar.System.__new__
+PyQt5.QtCore.QCborKnownTags.__new__
+PyQt5.QtCore.QCborSimpleType.__new__
+PyQt5.QtCore.QDateTime.YearRange.__new__
+PyQt5.QtCore.Qt.HighDpiScaleFactorRoundingPolicy.__new__
+PyQt5.QtGui.QColorSpace.Primaries.__new__
+PyQt5.QtGui.QColorSpace.TransferFunction.__new__
+PyQt5.QtGui.QTextBlockFormat.MarkerType.__new__
+PyQt5.QtNetwork.QOcspCertificateStatus.__new__
+PyQt5.QtNetwork.QOcspRevocationReason.__new__
+PyQt5.QtNetwork.QSslCertificate.PatternSyntax.__new__
+PyQt5.QtNetworkAuth.QAbstractOAuth.ContentType.__new__
+PyQt5.QtNetworkAuth.QAbstractOAuth.Error.__new__
+PyQt5.QtNetworkAuth.QAbstractOAuth.Stage.__new__
+PyQt5.QtNetworkAuth.QAbstractOAuth.Status.__new__
+PyQt5.QtNetworkAuth.QOAuth1.SignatureMethod.__new__
+PyQt5.QtNetworkAuth.QOAuth1Signature.HttpRequestMethod.__new__
+PyQt5.QtNfc.QNearFieldManager.AdapterState.__new__
+PyQt5.QtTest.QAbstractItemModelTester.FailureReportingMode.__new__
+PyQt5.QtWebEngineCore.QWebEngineUrlScheme.Syntax.__new__
+PyQt5.QtWebEngineWidgets.QWebEnginePage.LifecycleState.__new__
+PyQt5.QtWidgets.QActionGroup.ExclusionPolicy.__new__
```

Comment:
Regardless we're still waiting on the mypy/stubtest fix to be released...  but aside from that, what do you think of leaving these to be dealt with as part of #106?  Just to get the extra testing in sooner than later (maybe, if mypy gets a release sooner...).

Reply:
@BryceBeagle ^

Reply:
Sounds good to me